generated/nidaqmx/tests/test_channel_creation.py

```diff
@@ -0,0 +1,292 @@
+from nidaqmx import constants
```

Comment:
Delete this file.

Reply:
It exists in the root, so I think it just didn't get picked up as a rename for some reason?

Reply:
There should not be any tests in the `generated/` directory.

Reply:
D'oh, I totally spaced on the path!